modules/govuk_jenkins/manifests/config.pp

```diff
@@ -77,6 +85,8 @@
   $admins = [],
   $manage_config = true,
   $version = $govuk_jenkins::version,
+  $create_agent_role = false,
+  $jenkins_agent_user = $govuk_ci::agent::swarm::swarm_user,
```

Comment:
I agree with this. I was thinking about it and thought it felt backward to call that class from the main govuk_jenkins class.